purewheel/action.yml

```diff
@@ -0,0 +1,34 @@
+name: purewheel
```

Comment:
1. Inline this in .github/workflows/wheels-pure.yml
2. Consider using `container:` rather than `run: docker run ...`